api/napi/src/interpreter/diagnostic.rs

```diff
@@ -4,9 +4,13 @@
 use napi::bindgen_prelude::{FromNapiValue, ToNapiValue};
 use slint_interpreter::{Diagnostic, DiagnosticLevel};
 
+/// This enum describes the level or severity of a diagnostic message produced by the compiler.
 #[napi(js_name = "DiagnosticLevel")]
 pub enum JsDiagnosticLevel {
```

Comment:
This type is not visible in the generated documentation, even though it is referenced via the `CompileError` (new name :), because it's used as a property. However, it should be visible.

Reply:
done